src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/error/ErrorType.java

```diff
@@ -101,7 +101,12 @@ public enum ErrorType {
     /**
      * The request cannot be processed.
      */
-    UNPROCESSABLE_REQUEST("The request cannot be processed");
+    UNPROCESSABLE_REQUEST("The request cannot be processed"),
+
+    /**
+     * The error message for Y-Flow subflow that can not be rerouted since already on the best path.
+     */
+    ALREADY_ON_BEST_PATH("The sub-flow already on the best path");
```

Comment:
NOTE: this list of error is very generic, but this error description is very specific. 
I mean `ALREADY_ON_BEST_PATH` is good error, but `"The sub-flow already on the best path"` is only about sub flows, however `ALREADY_ON_BEST_PATH` could also be used by common flows. Maybe it's better to change description to `The flow already on the best path`?

Reply:
Thanks, fixed!